addon_config.mk

```diff
@@ -56,11 +324,30 @@ linux64:
 	# ADDON_INCLUDES += /opt/intel/mkl/include
 
 android/armeabi-v7a:
+	ADDON_CFLAGS += -Wno-strict-aliasing
+
+	# Exclude includes and source.
+	ADDON_SOURCES_EXCLUDE = libs/dlib/include/%
+	ADDON_INCLUDES_EXCLUDE = libs/dlib/include/%
+
 	ADDON_LIBS = libs/dlib/lib/android/armeabi-v7a/libdlib.a
 	# If your processor supports SIMD NEON instructions.
 	# ADDON_CPPFLAGS += -mfpu=neon
 
 android/x86:
+	ADDON_CFLAGS += -Wno-strict-aliasing
+
+	# Exclude includes and source.
+	ADDON_SOURCES_EXCLUDE = libs/dlib/include/%
+	ADDON_INCLUDES_EXCLUDE = libs/dlib/include/%
+	
 	ADDON_LIBS = libs/dlib/lib/android/x86/libdlib.a
 	# If your processor supports SIMD AVX instructions.
 	ADDON_CPPFLAGS += -mavx
+
+vs:
+	ADDON_DEFINES += DLIB_PNG_SUPPORT
+	ADDON_CFLAGS += /bigobj
```

Comment:
Seems like this is not working with latest PG.
But ADDON_CFLAGS = "-O3 /bigobj" works.

Also VS warning like unknown option -O3. So it might be okay without it.
ADDON_DEFINES does not work as you wrote in Getting started doc.

Reply:
You're right. The config already uses the ADDON_DEFINES += which only works in an updated version of the PG. So I would change this. The addon defines also will not work, but what part of the Getting started doc do you mean? The part where you should make the DLIB_PNG_SUPPORT entry manually?